docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -211,10 +224,10 @@ Alternatively, navigate to your Data storage directory and enter the following c
 [#CB90061]
 === Disk Write Queue (CB90061)
 
-*Background*: Having items in the disk write queue for too long could cause backoffs. 
+*Background*: Having items in the disk write queue for too long could cause backoffs.
 This usually indicates either a sizing issue, or a storage medium failing.
 
-*Condition*: If the time since the object most recently persisted to disk was added to the Disk Write Queue is over 50 seconds, a warning is produced. 
+*Condition*: If the time since the object most recently persisted to disk was added to the Disk Write Queue is over 50 seconds, a warning is produced.
```

Comment:
This reads very well but it may be worth adding additional information in the background to clarify with different less concise phrasing, e.g.,

"Each time an object is persisted to disk, the total time it spent in the Disk Write Queue is checked. If this is over 50 seconds, a warning..."

Reply:
Again same with above, vs code fixed the blank space at the end, would it be appropriate to fix in here or should I do it in a new branch?

Reply:
fixed